### DISCUSSION TITLE
SerialPort crate document in Crates.Io

### DISCUSSION THREAD

**Discussion by jerrywrice** [serialport/serialport-rs#93] [Q&A]
The top-level SerialPort crate document at https://crates.io/crates/serialport indicates the SerialPort cross-platform API is extended into platform specific functionality via TTYPort (Linux/Unix os) and COMPort (MS Windows) structs. I'm having difficulty finding the COMPort description of such in the rust documentation. Can someone please clarify where this might be found?

**Comment by eldruin** [comment]
That is because the docs are not built for windows on docs.rs. You can read the code for [COMPort here](https://github.com/serialport/serialport-rs/blob/9c0285ad5da6201aaf9a8b61e50178bc5d629c15/src/windows/com.rs#L29)
If you check out this repository on a windows machine and run `cargo doc --open`, you can read the docs for windows.
